tests/performance-tests/README.md

```diff
@@ -101,8 +101,8 @@ This step is only required if running performance tests that require access to t
 It requires the above step of creating environment-specific .env.json files first.
 
 ```bash
-export AUTH_DETAILS_AS_JSON=$(npm run log-auth-details --silent --environment=<environment> --users=<user name>)
-```
+npm run store-environment-details --environment=<environment> --users=<user names>
```

Comment:
just wondering if the `--users` accepts a comma seperated list of users based on this? i.e. `user1, user2, user3`

Reply:
That'll be the plan!